ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-intended-config.j2

```diff
@@ -88,6 +88,8 @@
 {% include 'eos/trackers.j2' %}
 {# router path-selection #}
 {% include 'eos/router-path-selection.j2' %}
+{# application traffic recognition #}
+{% include 'eos/application-traffic-recognition.j2' %}
```

Comment:
Have you checked the order of insertion here?

Reply:
it should be between interface Vxlan and monitor connectivty according to my EOS 4.31.1F ...